### PR TITLE
fix comments

### PR DESCRIPTION
**Pull request by Hliebov**

### CODE REVIEW

File: dirwatcher.js [hunk 1]
```diff
@@ -8,16 +8,16 @@ class DirWatcher {
 	}
 
 	watch(path, delay) {
-		let fsTimeout;
+		let fsTimeouts = [];
```
Comment (opasno) [inline dirwatcher.js]: It should be an object, not an array.

**Comment by opasno** [comment]
In `importer.js` on lines 15 and 28 you should be using `EventEmitter#once`, rather than `EventEmitter#on` to just listen for a single event dispatch.

**Comment by opasno** [comment]
Use `const` instead of `let` when you're not planning to reassign in future.

File: dirwatcher.js (hunk 1 above)
Comment (Hliebov) [reply dirwatcher.js]: yep, thanks, initially was another idea)

**Pull request merged**